app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -176,7 +176,7 @@
 
   StaticAnalytics.prototype.setThemesDimension = function(themes) {
     this.setDimension(3, themes || 'other');
-  };
```

Comment:
This was a revert of a change that included this fix. We will add the fix again independently.